### ISSUE TITLE
Identity function doesnt work

### ISSUE THREAD

**Issue by arunmahadevan** [hortonworks/streamline#836]
Apply `IDENTITY` function on a field in projection processor and save. Re-opening projection processor does not show previously saved data.

In the backend keyword IDENTITY throws a parse error with calcite.

`Caused by: org.apache.calcite.sql.parser.impl.ParseException: Encountered "IDENTITY" at line 1, column 8.
Was expecting one of:`

**Comment by priyank5485** [comment]
@arunmahadevan Is there a UI associated bug as well given that re-opening projection processor does not show previously saved data? Should i keep this open for that? I have merged your PR that has curl change.

**Comment by arunmahadevan** [comment]
Yes, in the projection processor, UI seems to be displaying the "name" rather than "displayName" for functions. @shahsank3t, can you take a look?

**Comment by shahsank3t** [comment]
@priyank5485 @arunmahadevan I'll take a look and make required changes today.
